app/views/sufia/admin/admin_sets/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<% content_for :page_header do %>
+  <h1><span class="fa fa-sitemap"></span> Administrative Set</h1>
```

Comment:
Given that you are internationalizing "Administrative Sets" , should this also be internationalized?